.dir-locals.el

```diff
@@ -84,6 +84,7 @@
   (clojure-indent-style . always-align)
   (cljr-favor-prefix-notation . nil)
   (cljr-insert-newline-after-require . t)
+  (cljr-print-right-margin . 118)
```

Comment:
ha, I love that emacs people have a shared config. maybe I need to start one for vim 😆

Reply:
nah, you should switch to emacs; it's a better vim than vim, y'know 😁